src/rules.jl

```diff
@@ -112,18 +111,24 @@ _abs_deriv(x) = signbit(x) ? -one(x) : one(x)
 
 # unary #
 #-------#
-
-@define_diffrule SpecialFunctions.erf(x)         = :(  (2 / sqrt(π)) * exp(-$x * $x)       )
+@define_diffrule SpecialFunctions.erf(x) = :( 2 * exp(-$x^2) * $invsqrtπ )
+@define_diffrule SpecialFunctions.erfinv(x) = :( (exp(SpecialFunctions.erfinv($x)^2) * $sqrtπ) / 2 )
+@define_diffrule SpecialFunctions.erfc(x) = :( -(exp(-$x * $x) * $invsqrtπ) * 2 )
+@define_diffrule SpecialFunctions.erfcinv(x) = :( -(exp(SpecialFunctions.erfcinv($x)^2) * $sqrtπ) / 2 )
+@define_diffrule SpecialFunctions.erfi(x) = :( 2 * exp($x * $x) * $invsqrtπ )
+@define_diffrule SpecialFunctions.erfcx(x) = :( 2 * ($x * SpecialFunctions.erfcx($x) - $invsqrtπ) )
```

Comment:
Are there any new rules? Or did you just move them around?

Reply:
Oh crap, I think these are repeated by a mistake when merging with the main branch or something.